src/aspire/commands/__init__.py

```diff
@@ -0,0 +1,9 @@
+import functools
```

Comment:
I don't really see much distinction, but I guess intent might be clearer in a `_config.py`

I wouldn't spend too much effort on this workaround; Because I think there is enough chatter that there will be a mainlined way to do this (global defaults) in CLICK that does not require heavier tooling like groups etc. 

This is actually in the spirit of one of those feature discussions. It is a monkey patch for sure.

Reply:
> I guess intent might be clearer in a `_config.py`

Yeah that was my thinking.

> Because I think there is enough chatter that there will be a mainlined way to do this (global defaults) in CLICK that does not require heavier tooling like groups etc.

Ok sounds good. We can leave this as is, then.